packages/contract-templates/package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "@0x/contract-templates",
```

Comment:
What about naming this `abi-gen-templates`? I like having all the `abi-gen` packages, grouped together when packages are sorted alphabetically, and `abi` alludes to `contracts`, so we don't really need to say it explicitly.